packages/fxa-payments-server/server/config/index.js

```diff
@@ -20,6 +20,13 @@ const conf = convict({
       env: 'FEATURE_USE_SCA_PAYMENT_UI_BY_DEFAULT',
       format: Boolean,
     },
```

Comment:
It looks like `useSCAPaymentUIByDefault` (the only other feature flag whose [patch](https://github.com/mozilla/fxa/commit/26266e48afbb20bc5cf6a35cca433666d0437aaa) I referenced for this patch) isn’t used anymore in `App.tsx`. Should it be removed? Happy to file a bug for that.

Reply:
In general, there's a [TODO comment for how this feature flag was added](https://github.com/mozilla/fxa/blob/c50d1175d522abd793cdb7daef0e43680c6d38d6/packages/fxa-payments-server/server/lib/server.js#L55)... -- how would this be improved, and when would be the appropriate time to add this improvement?

Reply:
Yeah, let's remove `useSCAPaymentUIByDefault` if it's not used.

As for feature flag improvements, it's not something we've discussed yet.  And we don't have any redis based feature flags afaik.